QgisModelBaker/tests/testdata/ilimodels/CHBase_Part4_ADMINISTRATIVEUNITS_20110830.ili

```diff
@@ -39,10 +40,10 @@ TYPE MODEL CHAdminCodes_V1 (en)
 END CHAdminCodes_V1.
 
 !! ########################################################################
-!!@technicalContact=models@geo.admin.ch
-!!@furtherInformation=http://www.geo.admin.ch/internet/geoportal/de/home/topics/geobasedata/models.html
+!!@technicalContact=mailto:models@geo.admin.ch

```

Comment:
What is the reason for these corrections? 
Would it fail with the old file?

Reply:
> What is the reason for these corrections?

The official model repository made such adjustments to the model. Probably they realized their model was invalid after an ili2c update. Same happened to a Colombian model.

>  Would it fail with the old file?

Yes, look at the Travis build after the second commit in this PR.

Reply:
Okay thanks for the info.